pulp_ansible/app/models.py

```diff
@@ -198,6 +209,9 @@ class Meta:
                 condition=Q(is_highest=True),
             )
         ]
+        permissions = [
+            ("manage_roles_collectionversion", "Can manage roles on collection versions")
+        ]
```

Comment:
We do not manage permissions on the content level, but on the repository level.
One reason is that content is shared between repositories and so multiple users can own the same content implicitly. Another (soft) reason is that Pulp expects to have _a lot_ of content, and this would blow up uncontrolled.